kep/9-capacity-scheduling/README.md

```diff
@@ -320,6 +321,241 @@ preemptor’s priority as potential victims in a node.
 We will watch the event of ElasticQuota and pod. The status of ElasticQuota 
 like allocated will store in Cache.
 
+### Additional Preemption Details
+Preemption happens when a pod is unschedulable, i.e., failed in PreFilter or Filter phases.
+
+In particular for capacity scheduling, the failure reasons could be:
+- Prefilter Stage
+  - sum(allocated res of pods in the same elasticquota) + pod.request > elasticquota.spec.max
+  - sum(allocated res of pods in the same elasticquota) + pod.request > sum(elasticquota.spec.min)
+
+So the preemption logic will attempt to make the pod schedulable, with a cost of preempting other running pods.
+
+#### ⚠️ Cross Node Preemption is not supported
+Current default Kubernetes scheduler does not support cross node preemption, which means that preemption process only happens in one node. In order to make capacity scheduler compatible with the default scheduler, current implementation also does not support cross node preemption.
+
+Because of that, it's expected in some cases the scheduler is incapable to give the resources back to the original namespace.
```

Comment:
```suggestion
Because of that, it's expected in some cases the scheduler is incapable to give the resources back to the original namespace. This will lead to resource fragmentation. In the production environment, it is recommended to set the sum of min to be less than the total resources of the cluster. This can avoid this problem as much as possible.
```